packages/fullstack/examples/cloudflare-workers-hello-world/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+use dioxus::prelude::*;
+use tracing_subscriber::prelude::*;
+use tracing_web::MakeConsoleWriter;
+
+#[cfg(feature = "server")]
+#[worker::event(start)]
+fn start() {
+    console_error_panic_hook::set_once();
+
+    let fmt_layer = tracing_subscriber::fmt::layer()
+        .with_ansi(true)
+        .without_time()
+        .with_writer(MakeConsoleWriter);
+    tracing_subscriber::registry().with(fmt_layer).init();
+
+    GetServerData::register_explicit().unwrap();
+    PostServerData::register_explicit().unwrap()
+}
+
+#[cfg(feature = "server")]
+#[worker::event(fetch)]
+async fn main(
+    req: worker::Request,
+    env: worker::Env,
+    _ctx: worker::Context,
+) -> worker::Result<worker::Response> {
+    handle_dioxus_application("/api/", req, env).await
+}
+
+pub fn app() -> Element {
+    let mut count = use_signal(|| 0);
+    let text = use_signal(|| "...".to_string());
+    // let server_future = use_server_future(get_server_data)?;
```

Comment:
If this is commented out, the following loop will enter an infinite loop. I could not figure out how to resolve this.

https://github.com/DioxusLabs/dioxus/blob/a3f8d1cd2cfe086373108416886d9967aa4fde9d/packages/web/src/lib.rs#L155

Reply:
#1926 fixes that issue

Reply:
Thanks, I'm going to try to make it work.